### PR TITLE
Capture session data

### PR DESCRIPTION
**Pull request by StephanieZuniga**

### CODE REVIEW

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m [hunk 1]
```diff
@@ -388,6 +388,27 @@ - (void)monitorAVPlayerItem {
                       forKeyPath:@"playbackBufferEmpty"
                          options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
                          context:MUXSDKAVPlayerItemPlaybackBufferEmptyObservationContext];
+        
+        AVAsset *asset = _player.currentItem.asset;
```
Comment (nidhik) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:392]: let's pull this out into its own method

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m [hunk 2]
```diff
@@ -388,6 +388,27 @@ - (void)monitorAVPlayerItem {
...
+        // Load Session Data from HLS manifest
+        __weak MUXSDKPlayerBinding *weakSelf = self;
+        [asset loadValuesAsynchronouslyForKeys:@[@"metadata"] completionHandler:^{
+            NSMutableDictionary *sessionData = [[NSMutableDictionary alloc] init];
+            for (AVMetadataItem *item in asset.metadata) {
+                NSString *keyString = (NSString *)[item key];
+                NSString *prefix = @"io.litix.data.";
```
Comment (nidhik) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:399]: Let's define the prefix as a `NSString *const`

**Review by nidhik — APPROVED** [review]
lgtm pending some minor comments

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m (hunk 1 above)
Comment (StephanieZuniga) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:392]: updated

Comment (StephanieZuniga) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m]: updated

**Review by nidhik — APPROVED** [review]

**Pull request merged**